assets/app/scripts/services/data.js

```diff
@@ -1194,12 +1202,19 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
     return new URI({protocol: protocol, hostname: hostPort}).toString();
   };
 
-  var CACHED_RESOURCE = {
+  // - *should* be hit frequently enough that the LRU does not kick out as we wont fetch again
```

Comment:
Will do. I think when I wrote the comment I was in a mindset of *hoping* :smile: